src/service/tables/governance_history.h

```diff
@@ -18,4 +18,11 @@ namespace ccf
   {
     static constexpr auto COSE_GOV_HISTORY = "public:ccf.gov.cose_history";
   }
+  using COSERecentProposals =
+    ServiceMap<std::string, std::string /* ProposalId */>;
```

Comment:
Assuming we don't care that this is a string (ie - don't manually mangle it or anything), we should use the existing typedef.

```suggestion
    ServiceMap<std::string, ccf::jsgov::ProposalId>;
```

Might need an additional include.

Reply:
Turns out we have a ccf::ProposalId _and_ a ccf::jsgov::ProposalId, so fixing that too.